worker/crates/action-processor/src/plateau/max_lod_extractor.rs

```diff
@@ -0,0 +1,155 @@
+use super::errors::PlateauProcessorError;
+use once_cell::sync::Lazy;
+use reearth_flow_runtime::{
+    channels::ProcessorChannelForwarder,
+    errors::BoxedError,
+    event::EventHub,
+    executor_operation::{ExecutorContext, NodeContext},
+    node::{Port, Processor, ProcessorFactory, DEFAULT_PORT},
+};
+use reearth_flow_types::{Attribute, AttributeValue, Feature};
+use regex::Regex;
+use serde_json::Value;
+use std::collections::HashMap;
+use std::path::Path;
+
+static DIGITS_PATTERN: Lazy<Regex> = Lazy::new(|| Regex::new(r"^\d+").unwrap());
+
+#[derive(Debug, Clone, Default)]
+pub struct MaxLodExtractorFactory;
+
+impl ProcessorFactory for MaxLodExtractorFactory {
+    fn name(&self) -> &str {
+        "PLATEAU.MaxLodExtractor"
+    }
+
+    fn description(&self) -> &str {
+        "Extracts maxLod"
+    }
+
+    fn parameter_schema(&self) -> Option<schemars::schema::RootSchema> {
+        None
+    }
+
+    fn categories(&self) -> &[&'static str] {
+        &["PLATEAU"]
+    }
+
+    fn get_input_ports(&self) -> Vec<Port> {
+        vec![DEFAULT_PORT.clone()]
+    }
+
+    fn get_output_ports(&self) -> Vec<Port> {
+        vec![DEFAULT_PORT.clone()]
+    }
+
+    fn build(
+        &self,
+        _ctx: NodeContext,
+        _event_hub: EventHub,
+        _action: String,
+        _with: Option<HashMap<String, Value>>,
+    ) -> Result<Box<dyn Processor>, BoxedError> {
+        let process = MaxLodExtractor {};
+        Ok(Box::new(process))
+    }
+}
+
+#[derive(Debug, Clone)]
+pub struct MaxLodExtractor {}
+
+impl Processor for MaxLodExtractor {
+    fn initialize(&mut self, _ctx: NodeContext) {}
+
+    fn num_threads(&self) -> usize {
+        5
+    }
+
+    fn process(
+        &mut self,
+        ctx: ExecutorContext,
+        fw: &mut dyn ProcessorChannelForwarder,
+    ) -> Result<(), BoxedError> {
+        let feature = &ctx.feature;
+
+        let package = feature.attributes.get(&Attribute::new("package")).ok_or(
+            PlateauProcessorError::DomainOfDefinitionValidator("package key empty".to_string()),
+        )?;
+
+        let city_gml_path = feature
+            .attributes
+            .get(&Attribute::new("cityGmlPath"))
+            .unwrap()
+            .clone();
+
+        let city_gml_path = city_gml_path.to_string();
+        let path = Path::new(&city_gml_path);
```

Comment:
```
use reearth_flow_common::uri::Uri
Uri::from_str(city_gml_path).unwrap().file_name()
```
It would be appreciated if the path could basically use reearth_flow_common::uri::Uri.

Reply:
Use `reearth_flow_common::uri::Uri.`
https://github.com/reearth/reearth-flow/pull/284/commits/3ed25010e15387159bfc85f05a575265f97b89f0